templates/tpus.libsonnet

```diff
@@ -28,13 +28,14 @@ local base = import 'base.libsonnet';
       metadata: {
         annotations: {
           "tf-version.cloud-tpus.google.com": tpuSettings.softwareVersion,
+	  "reserved.cloud-tpus.google.com": tpuSettings.reserved,
```

Comment:
I'm surprised this is different than the implementation for preemtible (ie with a resource name like `reserved-v3`). That would have made it easy to implement a limit with a ResourceQuota

Reply:
This also means that you could actually request a preemptible reserved TPU pod, which doesn't make any sense. Can you add a check to raise an error in that case?

Reply:
sure, I'll add an `assert`